src/app/admin/modules/data-management/components/data-graph/data-graph.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container-fluid">
+  <div class="row">
+      <div class="col-md-2">
+        <mat-form-field>
+          <mat-label>选择统计类型</mat-label>
+          <mat-select [formControl]="selectedGraphType">
+            <mat-option *ngFor="let statistics of statisticsType" [value]="statistics.id">{{statistics.name}}</mat-option>
```

Comment:
Please be consistent with variable names, you have `selectedGraphType`, but you use `statisticsType` here, it's confusing.